app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -9,7 +9,8 @@
     <p>You can chat to your dedicated adviser by phone, text or email.</p>
     <p>Find out more about <a href="/teacher-training-advisers">getting a teacher training adviser</a>.</p>
     <h3>Support from Now Teach</h3>
-    <p>If you have a degree, you may be able to get <a href="https://nowteach.org.uk/">support from the charity Now Teach</a>. They specifically help people get into teaching who are changing career.</p>
+    <p>Now Teach is a charity that provides support to career changers. Their service specifically helps experienced professionals get into teaching and offers two years of one-to-one career change support as they train and beyond.</p>
```

Comment:
Can we tweak slightly - it seems odd talking about `they` when normally we talk about `you`?